dist/bin/dotc

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
```

Comment:
@DarkDimius I agree there's a potential to merge the two. But I think we can delay the merge after the Copenhagen release -- as the `bin/dotc` and `bin/dotr` have something special to the development of the compiler, such as switch between bootstrap and unbootstrap versions. We can make the decision later.